tracer/src/Datadog.Trace/Ci/IntelligentTestRunnerClient.cs

```diff
@@ -453,16 +415,7 @@ private async Task<string[]> SearchCommitAsync(string[]? localCommits)
         async Task<string[]> InternalSearchCommitAsync(byte[] state, bool finalTry)
         {
             var request = _apiRequestFactory.Create(_searchCommitsUrl);
-            request.AddHeader(HttpHeaderNames.TraceId, _id);
-            request.AddHeader(HttpHeaderNames.ParentId, _id);
-            if (_useEvpProxy)
-            {
-                request.AddHeader(EvpSubdomainHeader, "api");
-            }
-            else
-            {
-                request.AddHeader(ApiKeyHeader, _settings.ApiKey);
-            }
+            SetRequestHeader(request, useApplicationHeader: true);
```

Comment:
```suggestion
            SetRequestHeader(request, useApplicationHeader: false);
```

Reply:
done in [b4dd46d](https://github.com/DataDog/dd-trace-dotnet/pull/3490/commits/b4dd46dabd3c99f8047cbb92342899da6df8f9e4)